lib/filtersRegistry.js

```diff
@@ -69,9 +68,16 @@ async function registerConfigFilters(nunjucks, templateDir, templateConfig) {
   const DEFAULT_MODULES_DIR = 'node_modules';
   if (!Array.isArray(confFilters)) return;
 
-  const promises = confFilters.map(async el => {
-    const modulePath = await isLocalTemplate(templateDir) ? path.resolve(templateDir, DEFAULT_MODULES_DIR, el) : el;
-    const mod = require(modulePath);
+  const promises = confFilters.map(async filtersModule => {
+    let mod;
+    try {
+      //first we try to grab module with filters by the module name
+      //this is when generation is used on production using remote templates
+      mod = require(path.resolve(DEFAULT_MODULES_DIR, filtersModule));
+    } catch (error) {
+      //in case template is local but was not installed in node_modules of the generator then we need to explicitly provide modules location
+      mod = require(path.resolve(templateDir, DEFAULT_MODULES_DIR, filtersModule));
+    }
```

Comment:
I think we must do the same in the hooksRegistry

https://github.com/derberg/generator/blob/7e94a395fcc8883271a9c82f0cab4e1aed504050/lib/hooksRegistry.js#L76-L77

Reply:
oh, that was a good call to get you on a PR. thanks so much, good 👁️ , done